java/main/src/main/java/com/saucelabs/saucebindings/citools/DefaultTool.java

```diff
@@ -0,0 +1,24 @@
+package com.saucelabs.saucebindings.citools;
+
+import com.saucelabs.saucebindings.SystemManager;
+
+public class DefaultTool implements CITool {
+
+  @Override
+  public String getToolName() {
+    String toolName = SystemManager.get("SAUCE_TOOL_NAME");
```

Comment:
What is `SAUCE_TOOL_NAME`? Do you mean the CI tool used?

Reply:
It's a unique key for the user to set a custom CI tool name if they want. Just needs something to fit the pattern. There might be a better name to make it more clear?